registry.py

```diff
@@ -1039,11 +1039,10 @@ def get_mapproxy(layer, seed=False, ignore_warnings=True, renderd=False):
 
     # The layer is connected to the cache
     layers = [
-        {
-            'name': layer_name,
-            'sources': ['default_cache'],
-            'title': str(layer.title),
-        },
+        {'name': layer_name,
+         'sources': ['default_cache'],
+         'title': "%s" % layer.title,
```

Comment:
yes, this catches a lot of ascii to utf problems - output is garbled in those cases though